css/css-positioning_absolute-no-advanced-requirements/css/styles.css

```diff
@@ -37,28 +37,49 @@ main {
 
 .box--top-left {
   background-color: #aed6f1;
+  position: absolute;
+  top: 0;
+  left: 0;
 }
 
 .box--top-center {
   background-color: #85c1e9;
+  position: absolute;
+  top: 0px;
+  left: 210px;
 }
 
 .box--top-right {
   background-color: #5dade2;
+  position: absolute;
+  top: 0;
+  right: 0;
 }
 
 .box--center {
   background-color: #3498db;
+  position: absolute;
+  top: 260px;
+  left: 210px;
 }
```

Comment:
You did it the calculating way. As we saw in the recap the better way would be to use transform: translate...

Reply:
jap